react/components/UI/MentionTextarea/queries/userSuggestions.js

```diff
@@ -0,0 +1,18 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  query MentionTextareaUserSuggestions($query: String!) {
+    suggestions: searches {
+      users: collaborators(query: $query, limit: 4, types: USER) {
+        ... on User {
+          id: slug
+          label: name
+        }
+        ... on Group {
```

Comment:
I don't think anything is set up to handle group mentions (though it's a good idea).

Reply:
Oh I know; I can delete this; this was to prevent the storybook from crashing because of the way mocks returns this.